device/samples/device_firmware_update_using_methods.js

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+'use strict';
+
+const { Client } = require('azure-iot-device');
+const { Mqtt: Protocol } = require('azure-iot-device-mqtt');
+const url = require('url');
+const { promisify } = require('util');
+const setTimeoutAsync = promisify(setTimeout);
+
+console.log('Initializing Device Client.');
+const client = Client.fromConnectionString(process.env.DEVICE_CONNECTION_STRING, Protocol);
+
+const reportFWUpdateThroughTwin = async (firmwareUpdateValue) => {
+    let patch = {
+        iothubDM : {
+            firmwareUpdate : firmwareUpdateValue
+        }
+    };
+    console.log(JSON.stringify(patch, null, 2));
+    let twin = await client.getTwin();
+    const asyncUpdateReportedTwinProperties = promisify(twin.properties.reported.update);
+    await asyncUpdateReportedTwinProperties(patch);
+};
+
+
+const applyImage = async (imageData) => {
```

Comment:
>imageData [](start = 26, length = 9)

What is the actual twin size limit these days?  What is a proposed new limit?

Reply:
I don't have answers to either of these questions. I don't know if the answers to these questions are included anywhere in previous samples, either.